Dockerfile

```diff
@@ -1,4 +1,4 @@
-ARG BASE_IMAGE=gradle:7.6.4-jdk11-alpine
+ARG BASE_IMAGE=gradle:7.6.4-jdk17-alpine
```

Comment:
We should upgrade gradle to latest in 3.0 if we haven't already

Reply:
I will try to make least impact for `2.x`. We can do major upgrades in v3.

Reply:
Agree!